modules/social_features/social_landing_page/social_landing_page.libraries.yml

```diff
@@ -30,11 +25,6 @@ section.block:
     theme:
       css/social_landing_page.section.block.css: {}
 
-featured:
-  css:
-    theme:
-      css/social_landing_page.featured.css: {}
-
```

Comment:
Removing this library is a backwards-incompatible change and may break other modules or themes overriding/extending the library. Ideally we mark the library as deprecated but keep using it for now.

Reply:
I did the same as you did [here](https://github.com/goalgorilla/open_social/pull/1766/files#diff-37d3a37e02d24dff66f6a231753c2c3d)

Reply:
Moving this one still need some work, as this change indeed breaks `node--featured.html.twig`, where we have a string `{{ attach_library('social_landing_page/featured') }}`.
So I think if we want to make `social_featured_content` fully decoupled we also need to move this library and all 
`--featured.html.twig` templates from `social_landing_page`, but not sure if to the `social_featured_content`, or just to the `socialbase` theme - to be consistent with other entity templates.
@Kingdutch what do you think?

Reply:
The templates are for the `featured` view mode so I think it depends:

1) Should this template only be used on landing pages, then we should probably renamed them so they only apply on landing pages. This requires us to update the `attach_library` call.
2) (more likely) If these templates are for any place where this view mode is used (which the name `node--featured` suggests, then it makes sense to move the template files to the module that provides those view modes.

My underlying philosophy for 2 and not moving to `socialbase` is that I think we have quite a few templates with crucial functionality in `socialbase` that should actually have been put in modules. The idea is that those modules should work even if the Open Social themes are not used. (That is not currently true)

Reply:
Since with this PR, we want to use Featured view mode not only on Landing Pages, but also on a Dashboard, the answer is 2 ;)
Currently, both templates and configs for featured view mode are still in `social_landing_page`, but if I understand correctly - we need to move them to this new `social_featured_content` - I will do that now.

Reply:
Ah okay, yeah I think the view mode needs to move too. Otherwise the `social_featured_content` module would have a dependency on `social_landing_page` because the featured-item paragraphs render in that view mode.

Reply:
I have moved all featured configs and templates in 18f7bbf and 79e213a